docs/administrator-documentation/moderne-platform/references/reporting.md

```diff
@@ -11,6 +11,12 @@ Moderne offers three types of reports for administrators:
 * **Audit logs** - every action any user takes in the system is logged. We currently have two ways of accessing it: via a UI, and via the API as a download in CEF format.
 * **Recipes run** - every recipe run by every user in your system. Includes the recipe name, time saved, user that ran it, recipe, as well as various statistics of the recipe run.
 * **Commits made** - every commit made by every user in your system. Includes the recipe run id (to be able to link the reports together), the type of commit, status of the commit job, as well as other statistics of the commit job.
+* **Usage** - every recipe run and associated commit by every user in your system. The generated report combines the commit and recipe run reports. This report includes 
+the recipe name, time saved, user that ran it, recipe, as well as various statistics of the recipe run. If any commit was created in association with the recipe run details
+of the commit are included: the type of commit, status of the commit job, as well as other statistics of the commit job. This report requires administrative permission to execute 
+* **Usage by organization** - every recipe run and associated commit by every user for a specific organization. The generated report combines the commit and recipe run reports. This report includes
+the recipe name, time saved, user that ran it, recipe, as well as various statistics of the recipe run. If any commit was created in association with the recipe run details
+of the commit are included: the type of commit, status of the commit job, as well as other statistics of the commit job.
```

Comment:
This doc exists under the "administrator documentation" section. Given that "usage by organization" is the only report that can be used by non-admins, maybe we move it to its own page under user documentation.

cc @mike-solomon  for doc organization -- do you think it makes sense to have some sort of "here's all the things you can do from an organization - devcenter, look at all repos, activity view, reports"?

Reply:
@kmccarp I feel like that's a question that you and people who interact with customers would have a better idea about.

Are non-admins confused about reporting or organizations? Are non-admins actively using those features? If so - then I think that would make sense to do.

Reply:
Reports were _only_ an admin function up until this point, which is why this doc was under administrator documentation. We are now adding a report that non-admins have access to (which an admin was requesting). That's why I am thinking out loud (and asking your opinion) on whether we should change the structure of the documentation around this.

Reply:
👍 fair - I was missing that context which is why I asked those questions. I feel like this doc should stay the same and include all of the reports. I then think we should make a separate user doc for reports available to general users in the user docs / moderne platform / how to guides.

Reply:
I have split out the  "usage by organization" report into a reports page in the docs / moderne platform / how to guides